src/main/java/io/debezium/connector/vitess/connection/VStreamOutputMessageDecoder.java

```diff
@@ -335,15 +342,37 @@ private void handleFieldMessage(Binlogdata.VEvent vEvent) {
     }
 
     private Table resolveTable(String schemaName, String tableName, List<ColumnMetaData> columns) {
+        List<String> pkColumnNames = new ArrayList<>();
```

Comment:
Also here it seems you prepare for multi-column PKs?

Reply:
We do support multi-column PKs, we also support single-column unique key, however, we don't support multi-column unique key. For example:

```
// Kafka message key = id + int_col
DROP TABLE IF EXISTS comp_pk_table;
CREATE TABLE comp_pk_table
(
    id             BIGINT NOT NULL,
    int_col        INT,
    int_col2        INT,
    PRIMARY KEY (id, int_col)
);

// Kafka message key = int_col3
DROP TABLE IF EXISTS no_pk_multi_comp_unique_keys_table;
CREATE TABLE no_pk_multi_comp_unique_keys_table
(
    id             BIGINT NOT NULL,
    int_col        INT,
    int_col2       INT,
    int_col3       INT,
    int_col4       INT,
    int_col5       INT,
    UNIQUE KEY unique_col_1_2 (int_col, int_col2),
    UNIQUE KEY unique_col_3 (int_col3),
    UNIQUE KEY unique_col_4_5 (int_col4, int_col5)
);
```

Reply:
I just added an integration test case for multi-column PK.

Reply:
Gotcha, thx.